packages/manager/modules/pci/src/projects/project/storages/databases/components/switch-price/switch-price.scss

```diff
@@ -0,0 +1,42 @@
+@import '@ovh-ux/ui-kit/dist/scss/tokens/_colors';
+
+$base-color: $p-500;
+$button-disabled: $p-075;
+
+@mixin buttonState($color: $base-color) {
+  border: 2px solid $button-disabled;
+  font-size: 0.9rem;
+  line-height: 0.9rem;
+  padding: 3px;
```

Comment:
**nitpick:** Could you please avoid writing custom css? I am aware that this screen is an exception, but it would be great if you can utilise the utility functions from bootstrap for `padding`, `font-size` and others.

Reply:
Done :)